log-collector-script/linux/eks-log-collector.sh

```diff
@@ -312,6 +312,7 @@ get_mounts_info() {
   lvs > "${COLLECT_DIR}"/storage/lvs.txt
   pvs > "${COLLECT_DIR}"/storage/pvs.txt
   vgs > "${COLLECT_DIR}"/storage/vgs.txt
+  cp --force /etc/fstab "${COLLECT_DIR}"/storage/fstab.txt
```

Comment:
what is `--force` for?

Reply:
There may not be any need for it. Just following the conventions already in place like the below command for collect `dmesg` logs:

`cp --force /var/log/dmesg "${COLLECT_DIR}/kernel/dmesg.boot"`

https://github.com/awslabs/amazon-eks-ami/blob/main/log-collector-script/linux/eks-log-collector.sh#L398